Expecto.Differ/Library.fs

```diff
@@ -0,0 +1,45 @@
+module Expecto.Differ
+open System
+open DiffPlex.DiffBuilder
+open DiffPlex.DiffBuilder.Model
+
+let colorizedDiff (colorizeText: ConsoleColor -> string -> string) first second =
+  let first, second =
+    match box first, box second with
+    | (:? string as f), (:? string as s) ->
+      string f, string s
+    | f, s ->
+      sprintf "%A" f, sprintf "%A" s
+
+  let differ = SideBySideDiffBuilder(DiffPlex.Differ())
```

Comment:
Is this object reusable? Hoist? Or is it stateful?

Reply:
Good question. I'm always unsure too when using non-functional first libraries. My guess is that the Builder has a side-effect and must be constructed every time.... But maybe the Differ() object is reusable?